packages/core/aqua-compiler/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@fluencelabs/aqua-compiler",
```

Comment:
I really dislike this new name for the library. Previous one was not the best. But this one is really misleading.

Reply:
Need other opinions

Reply:
`@fluencelabs/aqua-to-js` would be a lot better in my opinion

Reply:
Ok, i like it